release/models/system/openconfig-system.yang

```diff
@@ -701,6 +708,12 @@ module openconfig-system {
         "Indicates whether this server should be preferred
         or not.";
     }
+
+    leaf network-instance {
```

Comment:
I mentioned this in previous comments but the addition of this needs to take into account `/system/ntp/config/ntp-source-address` now.  If a server is reachable via an NI then any source-addressing needs to be addressed per server definition

Reply:
Is simply moving the source address down a level to the individual server config enough, or would more changes be necessary? Making that change, but can change it again!

Reply:
Leo has added source-address as a leaf in addition to network-instance as a leafref